### ISSUE TITLE
[LBL] Memory Issues + parallise inside compute/compile (or leave to APERO)?

### ISSUE THREAD

**Issue by njcuk9999** [njcuk9999/lbl#28]
For the FP it is worth parallelising inside LBL
For individual objects it is worth leaving to the user / apero

- [x] Create a way to run in parallel inside LBL (but default to 1 core)
- [ ] Test

**Comment by njcuk9999** [comment]
This is now done by passing an --iteration and --total key to lbl_computer.

Still todo: memory issues (move to new issue)
